exporters/keystone.go

```diff
@@ -40,115 +39,94 @@ func NewKeystoneExporter(client *gophercloud.ServiceClient, prefix string, disab
 	return &exporter, nil
 }
 
-func (exporter *KeystoneExporter) Describe(ch chan<- *prometheus.Desc) {
-	for _, metric := range exporter.Metrics {
-		ch <- metric.Metric
-	}
-}
-
-func ListDomains(exporter *BaseOpenStackExporter, ch chan<- prometheus.Metric) {
-
-	log.Infoln("Fetching domains information")
```

Comment:
No need to remove log lines.

Reply:
So the reason I removed it was because we pretty much have the same behavior here:

https://github.com/openstack-exporter/openstack-exporter/blob/master/exporters/exporter.go#L79

It's logging the same exact info pretty much?